src/kube-runtime/src/runtime

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
what if there's no `bash` in image?

Reply:
Ubuntu is our officially supported image, bash should be installed. Using /bin/sh may call dash shell and cause unexpected result.